### ISSUE TITLE
colspan

### ISSUE THREAD

**Issue by chadrien** [debois/elm-mdl#236]
First, thanks for this awesome project! 😺 

I have a question regarding tables: how can I apply a `colspan` to a `td` element?

**Comment by aforemny** [comment]
Are you talking about the `Material.Table.td` element? Does the following not work?

````
Material.Table.td [ Material.Options.Internal.attribute (Html.colspan 2) ] []
````

**Comment by chadrien** [comment]
Yeah I tried that already, and it *could* work, but not without a ugly hack which is editing the [`elm-package.json`](https://github.com/debois/elm-mdl/blob/7.6.0/elm-package.json) from `elm-mdl` as `Material.Options.Internal` is not in the `exposed-modules` list.

**Comment by vipentti** [comment]
This should be resolved in the next MINOR / MAJOR version with #213

**Comment by chadrien** [comment]
👍

Will wait for update then :)
